source/tests/system/nirfmxcdma2k_driver_api_tests.cpp

```diff
@@ -64,6 +72,334 @@ TEST_F(NiRFmxCDMA2kDriverApiTests, Init_Close_Succeeds)
+  EXPECT_SUCCESS(session, client::cfg_uplink_spreading(stub(), session, "", 0));
+  EXPECT_SUCCESS(session, client::select_measurements(stub(), session, "", MEASUREMENT_TYPES_CDA, true));
+  EXPECT_SUCCESS(session, client::cda_cfg_synchronization_mode_and_interval(stub(), session, "", CDA_SYNCHRONIZATION_MODE_SLOT, 0 , 1));
+  EXPECT_SUCCESS(session, client::cda_cfg_measurement_channel(stub(), session, "", 64, 0, CDA_MEASUREMENT_CHANNEL_BRANCH_I));
+  EXPECT_SUCCESS(session, client::cda_cfg_power_unit(stub(), session, "", CDA_POWER_UNIT_DB));
+  EXPECT_SUCCESS(session, client::initiate(stub(), session, "", ""));
+
+
+  const auto cda_fetch_code_domain_power_response = client::cda_fetch_code_domain_power(stub(), session, "", 10.0);
+  const auto cda_fetch_code_domain_i_and_q_power_response = client::cda_fetch_code_domain_i_and_q_power(stub(), session, "", 10.0);
+  const auto cda_fetch_symbol_evm_response = client::cda_fetch_symbol_evm(stub(), session, "", 10.0);
+  const auto cda_fetch_iq_impairments_response = client::cda_fetch_iq_impairments(stub(), session, "", 10.0);
+  const auto cda_fetch_code_domain_i_and_q_power_trace_response = client::cda_fetch_code_domain_i_and_q_power_trace(stub(), session, "", 10.0);
+  const auto cda_fetch_symbol_evm_trace_response = client::cda_fetch_symbol_evm_trace(stub(), session, "", 10.0);
+  const auto cda_fetch_symbol_constellation_trace_response = client::cda_fetch_symbol_constellation_trace(stub(), session, "", 10.0);
+
+  EXPECT_SUCCESS(session, cda_fetch_code_domain_power_response);
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.total_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.total_active_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.mean_active_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.peak_active_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.mean_inactive_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_power_response.peak_inactive_power()));
+  EXPECT_SUCCESS(session, cda_fetch_code_domain_i_and_q_power_response);
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_i_and_q_power_response.i_mean_active_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_i_and_q_power_response.q_mean_active_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_i_and_q_power_response.i_peak_inactive_power()));
+  EXPECT_TRUE(isnan(cda_fetch_code_domain_i_and_q_power_response.q_peak_inactive_power()));
+  EXPECT_SUCCESS(session, cda_fetch_symbol_evm_response);
+  EXPECT_TRUE(isnan(cda_fetch_symbol_evm_response.rms_symbol_evm()));
+  EXPECT_EQ(0.0, cda_fetch_symbol_evm_response.peak_symbol_evm());
+  EXPECT_TRUE(isnan(cda_fetch_symbol_evm_response.rms_symbol_magnitude_error()));
+  EXPECT_TRUE(isnan(cda_fetch_symbol_evm_response.rms_symbol_phase_error()));
+  EXPECT_TRUE(isnan(cda_fetch_symbol_evm_response.mean_symbol_power()));
+  EXPECT_EQ(0.0, cda_fetch_symbol_evm_response.frequency_error());
```

Comment:
For this `frequency_error` and the one for `chip_rate_error`, they failed on the `run_window_system_tests` step on the VM with actual values of -2400 and -13.1094, respectively. Can you run this one a few more times locally to see if it fails for you too? We may have to work on these assertions a little to avoid intermittent failures.

Reply:
@reckenro, thanks for your comments.  It is strange for the frequency error and chip error for that
 I ran the test locally about 20 more times, and no error is reported. Here I would like to use NEAR() to avoid intermittent failures.

Reply:
That makes sense. Minor but I think here and the rest of the spots in this file where you're doing `EXPECT_TRUE(NEAR...` you can instead just do [EXPECT_NEAR](https://google.github.io/googletest/reference/assertions.html#EXPECT_NEAR).

Reply:
Thanks. I have changed it to EXPECT_NEAR().